pkg/tasks/metricsserver.go

```diff
@@ -106,17 +106,19 @@ func (t *MetricsServerTask) create(ctx context.Context) error {
 			return fmt.Errorf("reconciling MetricsServer Service failed: %w", err)
 		}
 
-		cacm, err := t.factory.PrometheusK8sKubeletServingCABundle(map[string]string{})
+		kscm, err := t.client.GetConfigmap(ctx, "openshift-config-managed", "kubelet-serving-ca")
```

Comment:
Because the bundle isn't nominative (like the client certificate), the optimal setup would be to have a unique configmap created by a task from the first stage (like `NewMetricsClientCATask` or something similar).
However, for now, let's just add some comments here + around https://github.com/openshift/cluster-monitoring-operator/blob/a83f250fbf7efd2863a6faab471242322e563796/pkg/tasks/prometheus.go#L70 and around the two jsonnet resources, warning about the "duplication", to help avoid divergence between the two setups.